src/template-asciidoctor-node.js

```diff
@@ -0,0 +1,24 @@
+Opal = require('opal-runtime').Opal;
```

Comment:
shouldn't this line include a var?

```
var Opal = require('opal-runtime').Opal;
```

Reply:
Yes, nice catch!
I didn't know but it's actually a "feature" in JavaScript:

> Globals created with var (those created in the program outside of any function) cannot be deleted.
Implied globals created without var (regardless if created inside functions) can be deleted.

https://stackoverflow.com/a/6888838

Reply:
I know, it's very weird. In the future, we'll want to switch to `const`, which is even more restrictive..but that's kind of pointless right now given that Opal doesn't use `const` anywhere (because browser support still isn't universal).

Reply:
I switched to `const`. 
I think it should be possible to use flags in Opal to enable ECMAScript features. For instance if I want to target "ECMAScript 6 environment" then I should use `let` and `const` instead of `var` (where possible).

Reply:
I agree 1000%.